jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
+                sh 'oc delete all,template,daemonset -l jaeger-infra'
+                sh 'oc get all'
+            }
+        }
+        stage('all-in-one: deploy Jaeger') { 
+            steps { 
+                sh 'oc get all'
+                sh 'oc process -f all-in-one/jaeger-all-in-one-template.yml | oc create -f -'
+                sh 'oc get all'
+            }
+        }
+        stage('all-in-one: verify deployment'){
+            steps {
+                sh 'oc get dc'
+                openshiftVerifyDeployment apiURL: '', authToken: '', depCfg: 'jaeger', namespace: '', replicaCount: '1', verbose: 'true', verifyReplicaCount: 'true', waitTime: '3', waitUnit: 'min'
```

Comment:
Bonus points if you can get the images in a step of its own, so that we can wait "indefinitely" (or, say, 30 minutes) for the images, but have a timeout for our own parts.